src/data/full-quiz.js

```diff
@@ -53,6 +53,33 @@ const fullQuiz = [
     Explanation: "The maximum length for a file name is 255 bytes. The maximum combined length of both the file name and path name is 4096 bytes.",
     Link: "https://www.ibm.com/docs/en/spectrum-protect/8.1.9?topic=parameters-file-specification-syntax",
   },
+  {
+    Question: "In which of the following languages is function overloading not possible?",
+    Answer: "C",
+    Distractor1: "Python",
+    Distractor2: "Java",
+    Distractor3: "C++",
+    Explanation: "C is not an Object-Oriented Language,hence function overloading is not possible",
+    Link: "https://www.freecodecamp.org/news/object-oriented-programming-in-c/",
+  },
+  {
+    Question: "Which of the following Data Structure is used to convert an infix expression to postfix expression?",
+    Answer: "Stacks",
+    Distractor1: "Queues",
+    Distractor2: "Priority Queues",
+    Distractor3: "LinkedList",
+    Explanation: "Stack is most suitable data structure for converting an infix expression to postfix expression",
+    Link: "https://www.freecodecamp.org/news/understand-data-structures-in-c-and-cpp/",
+  },
+  {
+    Question: "Which of the following technique is implemented to improve CPU performance?",

```

Comment:
Proposed fix:

Question: "Which of the following techniques is implemented to improve CPU performance?",